skyportal/handlers/api/source.py

```diff
@@ -377,8 +378,22 @@ def get(self, obj_id=None):
             source_list[-1]["last_detected"] = source.last_detected
             source_list[-1]["gal_lon"] = source.gal_lon_deg
             source_list[-1]["gal_lat"] = source.gal_lat_deg
+            source_list[-1]["alias"] = ""  # placeholder until we get TNS names
             source_list[-1]["luminosity_distance"] = source.luminosity_distance
             source_list[-1]["dm"] = source.dm
+            source_list[-1]["saved_at"] = [
```

Comment:
I'm not sure what the utility of a list of datetimes is here decoupled from the actual groups they were saved to. I think the saved_at value should be added as a new key to each dict in source_list[-1]["groups"] after that's added on L401 below (will need to call to_dict() on each group beforehand)